code/modules/lootpanel/ss_looting.dm

```diff
@@ -0,0 +1,41 @@
+
+/// Slow search
+PROCESSING_SUBSYSTEM_DEF(looting)
+	name = "Loot Panel Search"
+	flags = SS_NO_INIT
+	runlevels = RUNLEVEL_LOBBY | RUNLEVELS_DEFAULT
+	wait = 0.5 SECONDS
+
+
+// Iterates over the tile contents for search objects without icons
+/datum/lootpanel/process(seconds_per_tick)
+	var/datum/tgui/panel = SStgui.get_open_ui(user, src)
+	if(isnull(panel))
+		stop_search()
+		return PROCESS_KILL
+
+	var/turf/tile = search_turf_ref?.resolve()
+	if(isnull(tile) || !user.TurfAdjacent(tile))
+		stop_search()
+		return PROCESS_KILL
+
+	var/processed = FALSE
+	for(var/ref in contents)
+		CHECK_TICK
```

Comment:
this is a very bad idea
it risks double processing or even just stacking sleeping procs into infinity

you need to create a subsystem manually that will take a return value from this process call that says "we're done" or "we aren't done"
then create a processing list on this item and clear it out as we handle shit